BranchSDK/Branch.m

```diff
@@ -540,6 +540,14 @@ + (void)setReferrerGbraidValidityWindow:(NSTimeInterval)validityWindow{
     }
 }
 
++ (void) setDMAParamsForEEA:(BOOL)eeaRegion AdPersonalizationConsent:(BOOL)adPersonalizationConsent AdUserDataUsageConsent:(BOOL)adUserDataUsageConsent{
+    @synchronized(self) {
```

Comment:
I know our old code uses this, but I doubt it does anything useful. It locks on the `[Branch class]` object. If anything, locking on the `[BNCPreferenceHelper sharedInstance]` would make more sense. But even that is likely unnecessary since the public methods in that class are synchronized on the instance.